programs/pkey/rsa_sign_pss.c

```diff
@@ -70,6 +73,13 @@ int main( int argc, char *argv[] )
     const char *pers = "rsa_sign_pss";
     size_t olen = 0;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  !  mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 137 and onwards, are displaying it without the negative. We should be self-consistent within the same program.